packages/qwik-city/buildtime/types.ts

```diff
@@ -14,7 +14,7 @@ export interface BuildContext {
 }
 
 export interface FrontmatterAttrs {
-  [attrName: string]: string;
+  [attrName: string]: any;
```

Comment:
why `any` Why is `string` not good enough? Can we do better then`any`?

Reply:
Because yaml accepts arrays (lists), objects (dictionaries), etc